app/Main.hs

```diff
@@ -263,8 +264,12 @@ configParser =
         metavar "OPT",
         help "GHC options to enable (e.g. language extensions)"
       ]
-    -- We cannot access cabal dependencies here
-    <*> pure []
+    <*> (many . strOption . mconcat)
+      [ long "package",
+        short 'p',
+        metavar "PACKAGE",
+        help "Explicitly specified dependency (for operator fixity/precedence only)"
+      ]
```

Comment:
I think that if we allow manual (CLI) configuration for the fixity map, it makes sense to allow users to specify the fixity of a single operator (not only packages), for example the fixity of an operator created by the user themself in their project. If we allow manual configuration, we also need a lot of the stuff I discarded from https://github.com/tweag/ormolu/tree/tbagrel1/fixity-with-config.

If we only add a `--package` option like you did in this commit, this option won't allow people using also `--cabal` to resolve a conflicting definition for an operator coming from cabal dependencies, because the manually specified packages will be at the same priority level as cabal dependencies (so that can only create more conflicts).

I know this commit is labelled as "WIP", but I just wanted to share some of the motivations/reasons about the amount of configuration I allow in https://github.com/tweag/ormolu/tree/tbagrel1/fixity-with-config